tests/cases/mount/mount.jsx

```diff
@@ -15,6 +15,44 @@ QUnit.module("mount()", {
     },
 });
 
+QUnit.test(
+    "with JSX and wrap",
+    assert =>
+    {
+        assert.expect(5);
+
+        let TestComponent = () =>
+        {
+            assert.ok(true, "render called");
+            return <div className="test"/>;
+        };
+
+        let fixture = findOne("#qunit-fixture");
+        fixture.innerHTML = `<div id="previous-sibling"></div><div id="container"></div><div id="next-sibling"></div>`;
+        mountJsxAndWrap("#container", TestComponent);
+
+        assert.strictEqual(
+            document.getElementById("qunit-fixture").children.length,
+            3,
+            "No sibling is destroyed"
+        );
+        assert.strictEqual(
+            document.getElementById("qunit-fixture").children[0].id,
+            "previous-sibling",
+            "Previous sibling is not moved"
+        );
+        assert.strictEqual(
+            document.getElementById("qunit-fixture").children[2].id,
+            "next-sibling",
+            "Next sibling is not moved"
+        );
+        assert.strictEqual(
```

Comment:
Can we also add an assertion that `children[1]` is wrapped in the `component-container` div?

Reply:
Yes 👍